tests/specs/api/forwarders_splunk.hurl

```diff
@@ -0,0 +1,56 @@
+GET http://localhost:5080/api/v1/forwarders
+Authorization: Bearer {{admin_token}}
+HTTP 200
+[Asserts]
+jsonpath "$.success" == true
+jsonpath "$.forwarders" count == 0
+
+PUT http://localhost:5080/api/v1/forwarders/splunk-test
+Authorization: Bearer {{admin_token}}
+Content-Type: application/json
+{
+  "forwarder": {
+    "version": 2,
+    "config": {
+      "type": "splunk",
+      "endpoint": "http://httpbin.org/anything",
+      "token": "test-token"
+    }
+  }
+}
+HTTP 200
+[Asserts]
+jsonpath "$.success" == true
+jsonpath "$.forwarder.config.type" == "splunk"
```

Comment:
`PUT` requests do not return the data, see `api/save_forwarder.go` :

https://github.com/link-society/flowg/blob/main/api/save_forwarder.go#L20-L22

Add a follow up `GET` request if you want to assert that the data has been saved correctly.